TelemetrySite/server/data_upload_scripts/data_upload.py

```diff
@@ -110,8 +119,12 @@ def handle_data(files):
                 cur.executemany(sql, dataToExecuteMany)
             except TimeoutError:
                 print(len(dataToExecuteMany))
+
     conn.commit()
     conn.close()
+    # remove csv files from folder
+    for file in files:
+        os.remove(file)
```

Comment:
Are you 100% sure the user is going to want these files to be deleted?

Reply:
This just delete the instance of the file saved by the program. The original file still exists on their computer or from where ever they uploaded it from, but the local copy saved by the server gets deleted to save space.